packages/api/package.json

```diff
@@ -26,7 +26,7 @@
     "dev": "concurrently \"yarn generate -w\" \"tsc --watch\" \"yarn dev:graphql\"",
     "dev:graphql": "nodemon --watch src --ext graphql --exec \"copyfiles \"src/**/*.graphql\" dist/esm\"",
     "dev:server": "tsx --no-cache ./local/server.ts",
-    "build": "graphql-codegen --config codegen.yml && (yarn build:cjs & yarn build:esm)",
+    "build": "graphql-codegen --config codegen.yml && (yarn build:cjs && yarn build:esm)",
```

Comment:
why didn't we have this before 🤔 
**question (non-blocking):** Now these commands will  run in sequence, right? if the build processes for CommonJS fails the the ESM build won’t run. How would this impact us?

Reply:
Yes, in sequence. This was fixed in our current version here
- https://github.com/vtex/faststore/pull/2361

So, I think this change won't impact v2 stores.